examples/hello_iced/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use iced::Element;
+
+fn main() -> iced::Result {
+    iced::program("Hello - Iced", HelloIced::update, HelloIced::view).run()
+}
+
+#[derive(Default)]
+struct HelloIced {}
+
+#[derive(Debug)]
+enum Message {}
```

Comment:
If your goal is a minimal Hello World instead of something others can start off of, you can elide both `HelloIced` and `Message` here and just use the unit type `()` for both. `update` and `view` can then be written as free functions
```rust
fn update(_: &mut (), _: ()) {}

fn view(_: &()) -> Element<()> { "Hello, Iced".into() }
```

Reply:
In my eyes, hello world is generally a starting point, some sort of template. You'd be stripping way too much off a "normal" Iced program that way. Just my two cents on it.

Reply:
Yes, I agree with @tsuza...I want this example to be more of a starting point, so I don't think it would be wise to strip the functions that much.